src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -309,39 +319,24 @@ private String getSubscriptionDetails(String newSubscription, String oldSubscrip
 
     private boolean isEventAllowedToProceedWithValidData(NotificationWrapper notificationWrapper,
                                                          NotificationEventType notificationType) {
-        if (REQUEST_INFO_INCOMPLETE.equals(notificationType)) {
-            if (StringUtils.isEmpty(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())
-                    || "No".equalsIgnoreCase(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())) {
-
-                log.info("Request Incomplete Information with empty or no Information From Appellant for ccdCaseId {}.", notificationWrapper.getNewSscsCaseData().getCcdCaseId());
-                return false;
-            }
+        if (REQUEST_INFO_INCOMPLETE.equals(notificationType)
+            && isNoOrNull(notificationWrapper.getNewSscsCaseData().getInformationFromAppellant())) {
```

Comment:
The behaviour here is slightly different than before. You might want to double-check it.

Besides the check for "No", the StringUtils.isEmpty checks both for `null` and an empty string `""`. This means that you are effectively treating an empty string as "not no".

What is supposed to happen if `getInformationFromAppellant` returns an empty string? Is it even possible?

Reply:
I thought isNoOrNull covered blanks, but that was in my YesNo Reactor (https://tools.hmcts.net/jira/browse/SSCS-10240) that got put on the backburner due to S&L 
That field should never be blank, only ever "Yes", "No" or null as per the ccd defs model, but to be safe ill change it to !isYes()